src/styles/primeng-theme/components/_datatable.scss

```diff
@@ -1,7 +1,10 @@
+@import "variables";
```

Comment:
Not sure if I'm missing something, but I don't think this file needs to have any changes here, right?